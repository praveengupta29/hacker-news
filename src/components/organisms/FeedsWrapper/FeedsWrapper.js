import React from 'react';

import Header from '../Header/Header';
import FeedList from '../../../containers/organisms/FeedList/FeedList';
import LoadMore from '../../../containers/atoms/LoadMore/LoadMore';

import './FeedsWrapper.scss';

const FeedsWrapper = () => {
  return (
    <div className="feeds-wrapper full-width">
      <Header />
      <div className="feed-list-load-more-container">
        <FeedList />
        <LoadMore />
      </div>
    </div>
  );
};

export default FeedsWrapper;
