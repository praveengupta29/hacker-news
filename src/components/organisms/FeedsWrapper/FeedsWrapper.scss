@import '../../../styles/utils/breakpoints';
@import '../../../styles/utils/colors';

.feeds-wrapper {
  background-color: $bg-color;
  margin: 0 auto;
}

.feed-list-load-more-container {
  padding: 24px 0;
}
