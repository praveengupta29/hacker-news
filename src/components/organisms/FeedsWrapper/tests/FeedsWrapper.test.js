import React from 'react';
import { shallow } from 'enzyme';
import FeedsWrapper from '../FeedsWrapper';

describe('FeedsWrapper component', () => {
  it('should render correctly and match snapshot', () => {
    const component = shallow(<FeedsWrapper />);
    expect(component).toMatchSnapshot();
  });
});
