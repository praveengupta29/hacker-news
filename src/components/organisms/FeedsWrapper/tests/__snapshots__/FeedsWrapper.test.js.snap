// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedsWrapper component should render correctly and match snapshot 1`] = `
<div
  className="feeds-wrapper full-width"
>
  <Header />
  <div
    className="feed-list-load-more-container"
  >
    <Connect(FeedList) />
    <Connect(LoadMore) />
  </div>
</div>
`;
